src/groupOverview/makeTable/addCustomTooltip.js

```diff
@@ -0,0 +1,114 @@
+import { select } from 'd3';
+
+/**
+ * Add custom tooltip functionality that allows interaction (e.g., clicking links).
+ * This replaces the native browser tooltip for specific cells.
+ *
+ * @param {object} cells - The cells of the table.
+ *
+ * @returns {void}
+ */
+export default function addCustomTooltip(cells) {
+    // Create tooltip container if it doesn't exist
+    let tooltip = select('body').select('.custom-tooltip');
+    if (tooltip.empty()) {
+        tooltip = select('body')
+            .append('div')
+            .attr('class', 'custom-tooltip')
+            .style('position', 'absolute')
+            .style('background', '#ffffff')
+            .style('color', '#000')
+            .style('padding', '8px 10px')
+            .style('border', '1px solid #ccc')
+            .style('border-radius', '3px')
+            .style('font-size', '11px')
+            .style('line-height', '1.3')
+            .style('white-space', 'pre-line')
+            .style('max-width', '350px')
+            .style('box-shadow', '0 2px 4px rgba(0,0,0,0.2)')
+            .style('z-index', '1000')
+            .style('pointer-events', 'auto') // Allow interaction with tooltip
+            .style('display', 'none')
+            .style('font-family', '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif');
+    }
+
+    // Filter Risk Score cells that need custom tooltips
+    const riskScoreCells = cells.filter((d) => d.column.valueKey === 'siteRiskScore' && d.tooltip);
+
+    riskScoreCells
+        .on('mouseenter', function(event, d) {
```

Comment:
There's some conflict between this and some other event listener.  Not sure if it's the row or the cell itself, but the tooltip disappears moving within the cell.